streams/src/main/java/org/apache/kafka/streams/processor/internals/PartitionGroup.java

```diff
@@ -84,6 +84,17 @@ RecordQueue queue() {
         streamTime = RecordQueue.UNKNOWN;
     }
 
+    long getPartitionTimestamp(final TopicPartition partition) {
+        return partitionQueues.get(partition).partitionTime();
+    }
+
+    void setPartitionTimestamp(final TopicPartition partition, final long timestamp) {
+        if (streamTime < timestamp) {
```

Comment:
Better to append suffix for time variables like `timestampMs`

Reply:
It's not a duration, hence, I am not sure if adding `Ms` suffix helps much here? `timestamp` is always UNIX epoch in ms throughout the whole code base and we never add `Ms` so far. Thoughts?